source/io/module_eggShell.js

```diff
@@ -112,8 +111,7 @@
         var Data_WriteSingle = Module.cwrap('Data_WriteSingle', 'void', ['number', 'number']);
         var Data_WriteDouble = Module.cwrap('Data_WriteDouble', 'void', ['number', 'number']);
         var EggShell_ExecuteSlices = Module.cwrap('EggShell_ExecuteSlices', 'number', ['number', 'number', 'number']);
-        var Occurrence_Set = Module.cwrap('Occurrence_Set', 'void', ['number']);
-        var OccurEvent = Module.cwrap('OccurEvent', 'void', ['number', 'number', 'number']);
+        var Occurrence_Set = Module.cwrap('Occurrence_Set', 'void', ['number']);6666
```

Comment:
Guessing the 6666 is a typo

Reply:
My keyboard is making a fool of me. Fixed this now